src/app/app.module.js

```diff
@@ -0,0 +1,39 @@
+(function (angular) {
```

Comment:
Please stop using IIFE: it is useless now that we are using webpack

Reply:
resolved in second commit